development/tools/jobs/kyma/components_test.go

```diff
@@ -95,11 +95,6 @@ var components = []struct {
 			jobsuite.JobFileSuffix("generic"),
 		},
 	},
-	{path: "permission-controller", image: tester.ImageGolangBuildpack1_16, suite: tester.NewGenericComponentSuite,
-		additionalOptions: []jobsuite.Option{
-			jobsuite.JobFileSuffix("generic"),
```

Comment:
Don't remove this test, and add this line
`jobsuite.Until(releases.Release124),`